tests/test_external_search.py

```diff
@@ -1684,12 +1749,13 @@ def assert_featured(description, worklist, facets, expect):
         #
         # The random element is relatively small, so it mainly acts
         # to rearrange works whose scores were similar before.
-        random_facets = FeaturedFacets(1, random_seed=41)
+        random_facets = FeaturedFacets(1, random_seed=43)
```

Comment:
I had to change the seed to get the same results as before. This doesn't make sense to me. The seed goes into Elasticsearch, and I don't know what happens in there.

Reply:
Weird!